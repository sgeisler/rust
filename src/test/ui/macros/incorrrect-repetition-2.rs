macro_rules! foo {
    ($($a:expr)*) => {};
    //~^ WARN `$a:expr` is followed (through repetition) by itself, which is not allowed for
}
