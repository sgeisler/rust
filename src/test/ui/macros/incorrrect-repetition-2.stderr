warning: `$a:expr` is followed (through repetition) by itself, which is not allowed for `expr` fragments
  --> $DIR/incorrrect-repetition-2.rs:2:8
   |
LL |     ($($a:expr)*) => {};
   |        ^^^^^^^ this fragment is followed by itself without a valid separator
   |
   = note: allowed there are: `;`, `=>` or `,`
help: add a valid separator for the repetition to be unambiguous, for example
   |
LL |     ($($a:expr);*) => {};
   |                ^

error[E0601]: `main` function not found in crate `incorrrect_repetition_2`
   |
   = note: consider adding a `main` function to `$DIR/incorrrect-repetition-2.rs`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0601`.
